src/Lesson_21/PersonReader.java

```diff
@@ -0,0 +1,55 @@
+package Lesson_21;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+    public class PersonReader {
+        static Scanner sc = new Scanner(System.in);
+        static ArrayList<String> arr = new ArrayList<>();
+
+        public static void reader() {
+            try {
+                String filePath = sc.next();
+                Reader reader = new FileReader(filePath);
+                BufferedReader bufferedReader = new BufferedReader(reader);
+                String i;
+                while ((i = bufferedReader.readLine()) != null) {
+                    System.out.print(i + "\n");
+                    arr.add(i);
```

Comment:
You parse file to the list of strings, but the task is to convert each line of the file into the `Person` object